BUILD.gn

```diff
@@ -915,6 +927,26 @@ dist_zip("electron_ffmpeg_zip") {
   ]
 }
 
+dist_zip("electron_native_mksnapshot_zip") {
+  data_deps = [
+    ":electron_mksnapshot",
+    ":licenses",
+  ]
+  outputs = [
+    "$root_build_dir/native_mksnapshot.zip",
+  ]
+}
+
+dist_zip("electron_mksnapshot_zip") {
```

Comment:
this is the exact same target as `electron_native_mksnapshot_zip`. I'd suggest dropping the `_native` one, and having the native mksnapshot build just build the same target but upload it under a different name.

Reply:
done!